timeStamp failures on items responses

Testing http://cite.opengeospatial.org/te2 (username `tomkralidis`, session `s0078`) against http://147.102.109.27:5001 yields 3 failures of type:

- `java.lang.AssertionError: timeStamp in response must not be after the request was send (2020-01-02T16:11:52Z) but was '2020-01-02T16:11:52.430407Z'`

--- a/src/main/java/org/opengis/cite/ogcapifeatures10/collections/FeaturesAssertions.java
+++ b/src/main/java/org/opengis/cite/ogcapifeatures10/collections/FeaturesAssertions.java
@@ -8,6 +8,7 @@ import static org.opengis.cite.ogcapifeatures10.util.JsonUtils.formatDate;
 import static org.opengis.cite.ogcapifeatures10.util.JsonUtils.hasProperty;
 import static org.opengis.cite.ogcapifeatures10.util.JsonUtils.parseAsDate;
 import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
 
 import java.net.URI;
 import java.net.URISyntaxException;
@@ -56,12 +57,7 @@ public class FeaturesAssertions {
                 return;
 
         ZonedDateTime date = parseAsDate( timeStamp );
-        assertTrue( !date.isAfter( timeStampAfterResponse.truncatedTo( SECONDS ) ),
-                    "timeStamp in response must not be after the request was send (" + formatDate( timeStampAfterResponse.truncatedTo( SECONDS ) )
-                                                             + ") but was '" + timeStamp + "'" );
-        assertTrue( !date.isBefore( timeStampBeforeResponse.truncatedTo( SECONDS ) ),
-                    "timeStamp in response must not be before the request was send (" + formatDate( timeStampBeforeResponse.truncatedTo( SECONDS ) )
-                                                             + ") but was '" + timeStamp + "'" );
+        assertNotNull(date, "Not valid timestamp.");
     }
 
     static void assertNumberReturned( String collectionName, JsonPath jsonPath, boolean skipIfNoNumberReturned ) {

